src/navigation/delivery/NewDeliveryPickupAddress.js

```diff
@@ -398,13 +403,4 @@ const styles = StyleSheet.create({
   },
 });
 
-function mapDispatchToProps(state) {
-  return {
-    country: state.app.settings.country.toUpperCase(),
-    store: selectStore(state),
-    deliveryError: state.delivery.assertDeliveryError,
-    addresses: state.delivery.addresses,
-  };
-}
-
-export default connect(mapDispatchToProps)(withTranslation()(NewDeliveryPickup));
+export default withTranslation()(NewDeliveryPickupAddress);
```

Comment:
That's not needed any more as `withTranslation()` is replaced by `  const { t } = useTranslation();` The same could (later) be done on other pages too